partner_email_check/models/res_partner.py

```diff
@@ -11,7 +11,7 @@
 try:
     from validate_email import validate_email
 except ImportError:
-    _logger.error('Cannot import "validate_email".')
+    _logger.debug('Cannot import "validate_email".')
```

Comment:
Please put info as other import errors.

Reply:
I asked my colleague to follow the conventions: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#importerror

Reply:
OK, I miss the discussion that leads to this convention, as I would prefer info for seeing it always when starting my Odoo service. Any way, let's keep it as convention says.

Reply:
thanks. I can't point to a discussion currently, but if I remember correctly the argument was that odoo's dependencies will disallow installing a module without the good libraries, and when debugging, you should turn on debug logging

Reply:
Yeah, makes sense.